content/ru/docs/concepts/overview/working-with-objects/finalizers.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Финализаторы
+content_type: concept
+weight: 80
+---
+
+# Финализаторы
+
+Финализаторы — это ключи в пространстве имен, которые сообщают Kubernetes дождаться выполнения определенных условий, прежде чем он полностью удалит ресурсы, помеченные для удаления. Финализаторы предупреждают контроллеры о необходимости очистить ресурсы, принадлежащие удаленному объекту.
+
+Когда вы указываете Kubernetes удалить объект, для которого указаны финализаторы, API Kubernetes помечает объект для удаления, заполняя .metadata.deletionTimestamp, и возвращает код состояния 202 (HTTP «Принято»). Целевой объект остается в завершающем состоянии, пока плоскость управления или другие компоненты выполняют действия, определенные финализаторами. После завершения этих действий контроллер удаляет соответствующие финализаторы из целевого объекта. Если поле Metadata.finalizers пусто, Kubernetes считает удаление завершенным и удаляет объект.
```

Comment:
- "Когда вы **указываете** Kubernetes удалить объект, для которого **указаны** финализаторы" — it's better to avoid the repeated verbs.
- "HTTP «Принято»" — we would prefer to leave the original "Accepted" term as something officially declared in the protocol/RFC.
- "плоскость управления" — we prefer to translate it as "управляющий слой".